src/redux/utilities.js

```diff
@@ -0,0 +1,41 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const appID = 'GahFETqP7Dmi6zLej4E7';
+
+export const addBook = createAsyncThunk('books/addBook', async (book) => {
+  try {
+    await axios.post(
+      `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${appID}/books`,
+      book,
+    );
+  } catch (e) {
+    throw new Error(e);
+  }
+  return book;
+});
```

Comment:
- [ ] Great job of working on the changes requested by the previous reviewer, However, after carefully reviewing your project I noticed the problem still persists. When a book is added for the first time it works fine but when you try adding a second or third book it fails to add. 
 - From your `PR` description I know you said the second reviewer @NahnahAJ was mistaken because everything is working fine from your side but am sorry to say she was right because am also having the same issue so kindly make sure to `fix` this issue before resubmitting for review.

- While tracing the error I noticed from the browser `network` tab that the response from the `API` is displaying a message saying `category' is required`. I have included a screenshot of the error below:
<img width="1329" alt="Screenshot 2023-08-23 at 18 09 19" src="https://github.com/katarighe/bookstore/assets/107192076/a5878b3f-8d63-4dfe-9703-98b12adbddd8">


Happy coding!😊🚀🍀